webapp-django/crashstats/supersearch/views.py

```diff
@@ -89,14 +96,29 @@ def get_params(request):
 
         params[key] = value
 
+    params['_sort'] = request.GET.getlist('_sort') or DEFAULT_SORT
     params['_facets'] = request.GET.getlist('_facets') or DEFAULT_FACETS
+    params['_columns'] = request.GET.getlist('_columns') or DEFAULT_COLUMNS
 
     allowed_fields = get_allowed_fields(request.user)
 
-    # Make sure only allowed fields are used
+    # Make sure only allowed fields are used.
+    params['_sort'] = [
+        x for x in params['_sort']
+        if x in allowed_fields or
+        (x.startswith('-') and x[1:] in allowed_fields)
+    ]
     params['_facets'] = [
         x for x in params['_facets'] if x in allowed_fields
     ]
+    params['_columns'] = [
+        x for x in params['_columns'] if x in allowed_fields
+    ]
+
+    # The uuid is always displayed in the UI so we need to make sure it is
+    # always returned by the model.
```

Comment:
Well "the uuid is always displayed in the UI" is the real explanation of it. I guess it could be slightly more precise by saying "the uuid is always displayed in the results in the UI", but I hope you don't mind if I don't bother. :)